assets/js/components/KeyMetrics/MetricsSelectionPanel/Footer.js

```diff
@@ -228,21 +228,22 @@ export default function Footer( { savedMetrics } ) {
 
 	return (
 		<footer className="googlesitekit-km-selection-panel-footer">
-			{ saveError && (
-				<ErrorNotice
-					error={ saveError }
-					noPrefix={ selectedMetrics?.length < 2 }
-				/>
-			) }
 			<div className="googlesitekit-km-selection-panel-footer__content">
-				<p className="googlesitekit-km-selection-panel-footer__metric-count">
-					{ sprintf(
-						/* translators: 1: Number of selected metrics, 2: Number of selectable metrics */
-						__( '%1$d of %2$d selected', 'google-site-kit' ),
-						selectedMetrics?.length || 0,
-						4
-					) }
-				</p>
+				{ ! saveError ? (
+					<p className="googlesitekit-km-selection-panel-footer__metric-count">
+						{ sprintf(
+							/* translators: 1: Number of selected metrics, 2: Number of selectable metrics */
+							__( '%1$d of %2$d selected', 'google-site-kit' ),
+							selectedMetrics?.length || 0,
+							4
+						) }
+					</p>
+				) : (
+					<ErrorNotice
+						error={ saveError }
+						noPrefix={ selectedMetrics?.length < 2 }
+					/>
```

Comment:
While looking further into this, the layout does also break in smaller viewports when the minimum selection requirement error appears. See:

![image](https://github.com/google/site-kit-wp/assets/20284937/0dcb5627-d41c-4c2e-b632-a5d4d2d1693f)

Can we improve the responsiveness here?

Reply:
@nfmohit Thanks, this seems to have slipped me. 

Updated. I ended up keeping the old layout on mobile, showing error above the content, and kept new layout on larger screens. Otherwise content would be too squeezed on mobile and look odd if we keep new layout, especially for languages other than English that can be wordy. Let me know what you think

Reply:
Thank you for the update, @zutigrm.

I do not think having a different composition of elements is a good idea in different devices. What do you think about just wrapping the metric count and actions one under another in small mobile devices (<450px)? Examples:

![image](https://github.com/google/site-kit-wp/assets/20284937/91052771-0c43-423e-b97c-064867286a73)

![image](https://github.com/google/site-kit-wp/assets/20284937/6086e909-2a4c-4796-b61c-9b670c4186ef)

![image](https://github.com/google/site-kit-wp/assets/20284937/4f41dfbb-ee80-48aa-91b5-3a35414a000d)

(the above experimentation uses a `flex-direction` of `column` and `row-gap` of `6px` for viewports under 450px in `.googlesitekit-km-selection-panel-footer__content`).

Reply:
@nfmohit Thanks for suggestion. I was considering between the two, but I went with the one that preserves the layout. This one makes more sense, I update it.